pkgs/applications/graphics/aview/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, aalib, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "aview";
+  version = "1.3.0rc1-debian4";
+
+  src = fetchFromGitHub {
```

Comment:
Why do you choose your _own_ fork for this? Upstream seems to point to sourceforge for source code download. If you'd like to advertise your own fork in NixOS, that's totally legit. Just name the package a little bit differently (maybe `aview-deepfire`) and change the homepage URL.

If your own fork in comparison with upstream is just a matter of a few bug fixes, it'd be a bit more pragmatic perhaps to use `patches` instead of a completely different fork.

Reply:
The project is dead and requires debian's patches to be applied -- which are the only changes I applied: https://github.com/deepfire/aview/commits/master

You're right about patches though, @doronbehar, that'd be the better option.